### PR TITLE
add crucial part of secure view prerequisites

### PR DESCRIPTION
**Pull request by voroyam**
the rich documents app version 2.2.0

fixes https://github.com/owncloud/docs/issues/2357

### CODE REVIEW

File: modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc [hunk 1]
```diff
@@ -8,34 +8,19 @@ These limitations include:
 * No downloading
 * No editing
 * Watermarking
-+
-NOTE: Administrators can configure the watermark text.
-* Optional printing and exporting to PDF with watermarks included
+* Optional printing and exporting to PDF with watermarks included, which can be adjusted
```
Comment (settermjd) [inline modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc]: Saying "Optional printing and exporting to PDF" seems redundant. What about just "Optional exporting to PDF..."?

File: modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc [hunk 2]
```diff
@@ -8,34 +8,19 @@ These limitations include:
...
-* OpenDocument Spreadsheet Document (.ods)
-* PDF
-
-If a folder shared with Secure View contains unsupported file types (e.g., JPG), they will not be accessible in any way.
-
 == Prerequisites
 
-* ownCloud 10.3 or higher _Enterprise Edition_
-* Collabora Online set up and integrated
+* ownCloud **10.3** or higher _Enterprise Edition_
```
Comment (settermjd) [inline modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc]: Do you want this bolded and italicised, or just bolded?

File: modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc [hunk 3]
```diff
@@ -8,34 +8,19 @@ These limitations include:
...
+* ownCloud Collabora Online App (richdocuments app) Version **2.2.0**
+* Collabora Online Server **4.0.10** set up and integrated
```
Comment (settermjd) [inline modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc]: Do you want these bolded and italicised, or just bolded?

File: modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc [hunk 4]
```diff
@@ -62,3 +47,17 @@ When "_{secure-view-label}_" is enabled, any attempts to download the file will
...
+
+* Microsoft Word (.docx)
+* Microsoft Excel (.xlsx)
+* Microsoft PowerPoint (.pptx)
+* OpenDocument Text Document (.odt)
+* OpenDocument Presentation Document (.odp)
+* OpenDocument Spreadsheet Document (.ods)
+* PDF
+
+If a folder shared with Secure View contains unsupported file types (e.g., JPG), they will not be accessible in any way.
```
Comment (settermjd) [inline modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc]: How about removing "in any way" at the end, as "will not be accessible" should be sufficiently explicit?

File: modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc (hunk 1 above)
Comment (voroyam) [reply modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc]: It was you who wrote this, now you say it's redundant?

This is the admin manual - so for admins. I expect that is known that everything here is for admins.

File: modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc (hunk 3 above)
Comment (voroyam) [reply modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc]: just bold

File: modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc (hunk 4 above)
Comment (voroyam) [reply modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc]: again, you wrote this, the only change I made was in the prerequisites.

File: modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc (hunk 3 above)
Comment (settermjd) [reply modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc]: Thanks.

File: modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc (hunk 4 above)
Comment (settermjd) [reply modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc]: Thanks.

**Pull request merged**